server/db/seeds.js

```diff
@@ -18,13 +18,23 @@ db.accounts.insertMany([
 db.topics.insertMany([
     {
         title: "animals",
-        word_list: ["tiger", "dog", "cat", "koala", "chicken", "turtle", "whale", "owl", "rooster", "pigeon", "zebra", "fox"],
+        word_list: ["shark", "dog", "cat", "koala", "otter", "turtle", "whale", "owl", "wolf", "walrus", "fish", "fox"],
```

Comment:
Can you pull 2 names out of this so there are only 10, just so the progress bar works

Reply:
sorry Ali forgot to but will do that right now

Reply:
no worries, i dont know if theres a way for me to do it from github but i couldnt see one